Sources/ArgumentParser/Utilities/SequenceExtensions.swift

```diff
@@ -9,14 +9,9 @@
 //
 //===----------------------------------------------------------------------===//
 
-extension Sequence where Element: Equatable {
-  func uniquified() -> [Element] {
-    var sequence = Array<Element>()
-    for element in self {
-      if !sequence.contains(element) {
-        sequence.append(element)
-      }
-    }
-    return sequence
+extension Sequence where Iterator.Element: Hashable {
```

Comment:
Is there a reason to switch from `Element` to `Iterator.Element` in this patch?

Reply:
@airspeedswift nothing specific and probably just an oversight on my part - happy to make the change back